pwndbg/chain.py

```diff
@@ -32,9 +32,9 @@ def get(address, limit=LIMIT, offset=0, hard_stop=None, hard_end=0):
     Returns:
         A list representing pointers of each ```address``` and reference
     """
-    limit = int(limit)+1
+    limit = int(limit)
     
-    result = []
+    result = [address]
```

Comment:
Regardless of what the limit value is, the chain should always start with the given address.